monkestation/code/modules/blood_for_the_blood_gods/slasher/slasher_outfit/slasher_clothing.dm

```diff
@@ -41,8 +62,26 @@
 	. = ..()
 	ADD_TRAIT(src, TRAIT_NODROP, "slasher")
 
-/obj/item/clothing/under/color/random/slasher
+/obj/item/clothing/suit/apron/slasher2
+	name = "butcher's apron"
+	desc = "A brown butcher's apron, you can feel an aura of something dark radiating off of it."
+	resistance_flags = INDESTRUCTIBLE | LAVA_PROOF | FIRE_PROOF | UNACIDABLE | ACID_PROOF
+	icon = 'monkestation/icons/mob/slasher/slasher_items.dmi'
+	worn_icon = 'monkestation/icons/mob/slasher/slasher_wornstates.dmi'
+	icon_state = "slasher_apron"
+	inhand_icon_state = null
+
+/obj/item/clothing/suit/apron/slasher2/Initialize(mapload)
+	. = ..()
+	ADD_TRAIT(src, TRAIT_NODROP, "slasher")
+
+/obj/item/clothing/under/slasher
 	name = "butcher's jumpsuit"
+	icon = 'monkestation/icons/mob/slasher/slasher_items.dmi'
+	worn_icon = 'monkestation/icons/mob/slasher/slasher_wornstates.dmi'
+	icon_state = "slasher_under"
+	inhand_icon_state = null
+	desc = "A brown butcher's jumpsuit, you can feel an aura of something dark radiating off of it."
 	clothing_traits = list(TRAIT_NODROP)
```

Comment:
`clothing_traits` applies the traits to the _mob wearing it_, not to the clothing itself